pkg/integrations/v2/ebpf/common.go

```diff
@@ -0,0 +1,43 @@
+package ebpf
```

Comment:
I'd prefer avoiding defining integrations twice. Is there a reason we need a v1 and v2 version of this integration?

Reply:
I've used the NamedShim to tie this into a v2 integration, I think it should be fine. I'll check that everything works as expected once more, just to be sure.